src/contracts/whoopi/types.ts

```diff
@@ -1,4 +1,5 @@
 import { ContractTransaction } from '@ethersproject/contracts';
+import { BigNumberish } from '@ethersproject/bignumber';
```

Comment:
we should be using this type instead: https://github.com/ethers-io/ethers.js/blob/ba967875f71019d367606f2b0bfae27879723a57/src.ts/utils/maths.ts#L19

it's because ethersproject/bignumber ^5.0.0 must be using `BigNumber` behind the scenes anyhow